code/modules/mob/living/carbon/human/human.dm

```diff
@@ -690,6 +690,10 @@
 	else if (href_list["lookitem"])
 		var/obj/item/I = locate(href_list["lookitem"])
 		usr.examination(I)
+	else if (href_list["listitems"])
+		var/obj/item/I = locate(href_list["listitems"])
```

Comment:
Because I have zero experience with hrefs and just attempted to copy how the "Take a closer look..." feature for item showing currently works.